doc/source/report/plots/plot_distribution.py

```diff
@@ -0,0 +1,14 @@
+import matplotlib.pyplot as plt
+from mousestyles. import data
+from mousestypes.est_power_param import fit()
```

Comment:
@changsiyao Since I need the estimator results from my teammates @z357412526 and it may take a while to generate the plots, I just uploaded the plots directly.